internal/helm/chart/dependency_manager.go

```diff
@@ -257,6 +259,11 @@ func (dm *DependencyManager) resolveRepository(url string) (repo repository.Down
 	defer dm.mu.Unlock()
 
 	nUrl := repository.NormalizeURL(url)
+	supported := repository.IsDepSupported(nUrl)
+	if !supported {
+		err = fmt.Errorf("unsupported chart repository URL: %s", nUrl)
+		return
+	}
```

Comment:
Can we provide a better reason here by making `IsDepSupported` return e.g. a descriptive error? I think this will reduce support questions by explicitly telling the user it's due to an alias.